aws/requests.yaml

```diff
@@ -25,6 +25,9 @@ releases:
         - name: chart-operator
           version: ">= 2.12.0"
           issue: https://github.com/giantswarm/giantswarm/issues/15213
+        - name: net-exporter
```

Comment:
I think we should put this into it's own bracket which is higher than the currently latest version.

Reply:
Changed let me know if this is better.